mw-webapp/src/router/pages.tsx

```diff
@@ -36,6 +37,10 @@ export const pages = {
     path: "users",
     element: <AllUsersPage />,
   },
+  redirect: {
+    path: "redirect",
```

Comment:
A noun to describe entities is better than verbs:
```suggestion
    path: "redirectPage",
```

Reply:
done